src/html.jl

```diff
@@ -206,3 +206,32 @@ function notebook2html(path::AbstractString)
     return notebook2html(notebook)
 end
 
+"""
+NOTE: NEW API for SessionActions support
+Run the `notebook` and return the code and output as HTML.
+"""
+function notebook2html2(
+        notebook::Notebook;
+        code_class="language-julia",
+        output_class="code-output",
+        hide_md_code=true
+    )
+
+    cells = [last(e) for e in notebook.cells_dict]
+    order = notebook.cell_order
+    outputs = map(order) do cell_uuid
+        cell = notebook.cells_dict[cell_uuid]
+        _cell2html(cell, code_class, output_class, hide_md_code)
+    end
+    return join(outputs, '\n')
+end
+
+"""
+NOTE: NEW API for SessionActions support 
+    notebook2html(path::AbstractString)
+
+Run the Pluto notebook at `path` and return the code and output as HTML.
+"""
+function notebook2html2(notebook::Notebook)
+    return notebook2html2(notebook)
+end
```

Comment:
```suggestion
```

This method didn't do anything because the one above is based on keyword arguments. No need to define the extra method:

```
julia> function foo(x; y=3)
             return x * y
         end;

julia> foo(2) = 6
```

Reply:
yaa makes sense. This method is useless. I should have cleaned up. By mistake pushed the one I hacking :(
Should fix this and create another pull request

Reply:
You can just update this PR. No need to make a new one, I think